boxoffice/views/admin_item_collection.py

```diff
@@ -59,3 +62,54 @@ def admin_item_collection(item_collection):
     return dict(title=item_collection.title, item_collection=item_collection, date_item_counts=date_item_counts,
         date_sales=date_sales, today_sales=today_sales,
         sales_delta=sales_delta(g.user.timezone, item_ids))
+
+
+def jsonify_new_item_collection(item_collection_dict):
+    ic_form = ItemCollectionForm()
+    if request.method == 'GET':
+        return jsonify(form_template=render_form(form=ic_form,
+            title=u"New item collection", submit=u"Create", ajax=False, with_chrome=False))
+    if ic_form.validate_on_submit():
+        ic = ItemCollection(organization=item_collection_dict['organization'])
+        ic_form.populate_obj(ic)
+        if not ic.name:
+            ic.make_name()
+        db.session.add(ic)
+        db.session.commit()
+        return api_success(result={'item_collection': dict(ic.current_access())}, doc=_(u"New item collection created"), status_code=201)
+    return api_error(message=_(u"There was a problem with creating the item collection"), errors=ic_form.errors, status_code=400)
+
+
+@app.route('/admin/o/<org>/ic/new', methods=['GET', 'POST'])
+@lastuser.requires_login
+@render_with({'text/html': 'index.html.jinja2', 'application/json': jsonify_new_item_collection})
+@load_models(
+    (Organization, {'name': 'org'}, 'organization'),
+    permission='org_admin'
+    )
+def admin_new_ic(organization):
+    return dict(organization=organization)
+
+
+def jsonify_edit_item_collection(item_collection_dict):
+    item_collection = item_collection_dict['item_collection']
+    ic_form = ItemCollectionForm(obj=item_collection)
+    if request.method == 'GET':
+        return jsonify(form_template=render_form(form=ic_form,
+            title=u"Edit item collection", submit=u"Save", ajax=False, with_chrome=False))
```

Comment:
This may be a good time to expose Baseframe's [FormGenerator](https://github.com/hasgeek/baseframe/blob/6c94f5b07eb279519a0d8ffe3c183df42ab1e82c/baseframe/forms/form.py#L128-L132) as an API. (Separate ticket)